src/app/frontend/filters_info.py

```diff
@@ -0,0 +1,28 @@
+import streamlit as st
+
+EMPTY_FILTERS = "No filters applied"
+NON_EMPTY_FILTERS = "Filters applied -\n\n"
+
+
+def show_filters_info(applied_filters: dict) -> None:
+    """Display information about applied filters.
+
+    Parameters
+    ----------
+    applied_filters : dict
+        A dictionary containing the applied filters, where keys are the filter descriptions and values represent the corresponding filter values.
+
+    Returns
+    -------
+    None
+    """
+    # No filters applied
+    if not applied_filters:
+        filters_info = EMPTY_FILTERS
+    else:
+        applied_filters_str = "\n\n".join(
+            [f"{key}: {value}" for key, value in applied_filters.items()]
+        )
+        filters_info = NON_EMPTY_FILTERS + applied_filters_str
```

Comment:
Some suggestion here, basically set default and modify if available, save on the else since the if case is only one line.

```suggestion
    filters_info = EMPTY_FILTERS
    
    # Filters applied
    if applied_filters:
        applied_filters_str = "\n\n".join(
            [f"{key}: {value}" for key, value in applied_filters.items()]
        )
        filters_info = NON_EMPTY_FILTERS + applied_filters_str
```